src/Person.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ name, age, sex, isMarried, partnerName }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${name}`}</h2>
+    {age && (<p className="Person__age">{`I am ${age}`}</p>)}
+    <p className="Person__partner">
+      {(isMarried === true)
```

Comment:
```suggestion
      {isMarried
```
Там и так тру или false